README.md

```diff
@@ -1,2 +1,221 @@
-# milk-wars
-Isn't the repository name descriptive enough?
+# Milk Wars
+
+A plugin for a CTF-like PVP event for Geek NZ.
+
+Geek NZ is a New Zealand-based Minecraft SMP server. From the [minecraft-mp page](https://minecraft-mp.com/server-s13774):
+
+> Our small New Zealand based server has always been popular with friends and family. We run a vanilla server on PaperMC with a few small additions to make life a little easier. If you are a nice person you are welcome to join us.
+
+<!--
+####################
+# TABLE OF CONTENTS
+####################
+-->
+# Table of contents
+
+* [Introduction](#introduction)
+* [Usage](#usage)
+    * [For players](#for-players)
+        * [Capturing the other clan's treasure](#capturing-the-other-clans-treasure)
+        * [Preventing the other clan from capturing your treasure](#preventing-the-other-clan-from-capturing-your-treasure)
+        * [Getting kills](#getting-kills)
+    * [For clan leaders](#for-clan-leaders)
+        * [Managing your clan members](#managing-your-clan-members)
+        * [Managing your clan treasure](#managing-your-clan-treasure)
+    * [For admins](#for-admins)
+        * [Joining and leaving a clan](#joining-and-leaving-a-clan)
+        * [Promoting and demoting clan leaders](#promoting-and-demoting-clan-leaders)
+        * [Changing chat colours](#changing-chat-colours)
+        * [Creating smart signs](#creating-smart-signs)
+* [Contribution](#contribution)
+
+<!--
+###############
+# INTRODUCTION
+###############
+-->
+# Introduction
+
+Milk Wars is mostly a Capture The Flag (CTF) like event between two clans, the Milk Drinkers and the Wool Wearers. Internally, and in commands, these are referred to as `cows` and `sheep`. Clan members are able to steal the opposite clan's treasure from their clan hall and bring it back to theirs for points. You can only do this once per day to prevent farming. But good luck finding the clan halls! They're hidden and may even be booby-trapped. The only way to identify a clan hall's location is the *clan tower* on it that's identical for both clans.
+
+The other aspect of Milk Wars is the PVP. Players can kill opposite clan members *Only if a chooses, by being signed in to their clan.* If they are signed out PVP is prohibited, so those not wishing to participate in the PVP can opt out of it. If you choose to participate however, check over your shoulders once in a while because you could get killed anywhere, anytime!
```

Comment:
```suggestion
The other aspect of Milk Wars is the PVP. Players can kill opposite clan members *Only if they are signed in to their clan.* If they are signed out PVP is prohibited, so those not wishing to participate in the PVP can opt out of it. If you choose to participate however, check over your shoulders once in a while because you could get killed anywhere, anytime!
```